.env.example

```
@@ -1,5 +1,10 @@
-# Environment variable template for dealdocs (Vite/Supabase)
-# Copy this file to .env and fill in your actual values
+# Environment variable template for dealdocs (Vue 3 + Supabase)
+# Copy this file to .env.local and fill in your actual values
+
+# Supabase Configuration (REQUIRED)
+VITE_SUPABASE_URL=your_supabase_project_url
+VITE_SUPABASE_ANON_KEY=your_supabase_anon_key
+VITE_SUPABASE_SERVICE_KEY=your_supabase_service_key
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_SUPABASE_SERVICE_KEY key should go before the VITE_SUPABASE_URL key

(UnorderedKey)